locales-pending/premium.ftl

```diff
@@ -156,10 +156,10 @@ modal-exposure-type-title = About exposure types
 # $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be more than 1.
 modal-exposure-type-description = We search for you in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info. Here are the two types of exposures we find:
 modal-exposure-type-data-breach = <b>Data breach</b> means your information has been compromised in a breach and could be in the wrong hands.
-  Resolving these typically requires accessing your accounts, so you’ll need to take manual steps to resolve each breach even if you’ve upgraded to { -brand-premium }.
+  Resolving these typically requires accessing your accounts, so you’ll need to take manual steps to resolve each breach even if you’ve upgraded to { -brand-monitor-plus }.
```

Comment:
Q: Should this "upgraded" be "subscribed"?

Reply:
It is correct per Figma, I've commented there to ask if that's correct.